tensorboard/plugins/profile/overview_page/overview-page.html

```diff
@@ -0,0 +1,353 @@
+<!--
+@license
+Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+   -->
+
+<!--
+  The Overview Page is a view within the tf-profile-dashboard.
+  It is the first page that users would look at in order to under
+  the overall performance of their TPU workloads. The page has five
+  sections of results:
+     (1) Performance summary
+     (2) Step-time graph
+     (3) Top 10 TensorFlow Operations executed on TPU
+     (4) Run environment
+     (5) Recommendation for next steps
+-->
+
+<link rel="import" href="../vz-line-chart/vz-line-chart.html">
+<link rel="import" href="../paper-card/paper-card.html">
+
+<dom-module id="overview-page">
+  <template>
+    <style>
+     paper-card {
+       margin: 5px;
+       --paper-card-header-color: white;
+       --paper-card-header: background-color: blue;
+     }
+     .steptime-average {
+       font-weight:bold;
+       font-style:italic;
+       color:red;
+     }
+     .table-style {
+       table-layout:auto;
+       width:95%;
+     }
+     .top-ops-table {
+       width:100%;
+     }
+     .x-axis-title {
+       width:80%;
+       text-align:center;
+       text-transform: capitalize;
+     }
+     .y-axis-title {
+       width:10px;
+       writing-mode:bt-rl;
+       transform: rotate(270deg);
```

Comment:
nit: Be consistent about whether to have a space before the value. Maybe add a space before all values.

Reply:
Fixed.